js/feed.js

```diff
@@ -1,141 +1,213 @@
+            iconColor = "#24292e";
+            content = ""; // undefined before and altered content message
+            field.payload.commits.map(function (commit) {
+              content += commit.message + "<br>";
+            })
+            link = field.payload.compare;
+            break;
+        }
+
+        if (!eventType)
+          return; // don't display if event not handled above
+
+        $(".feed-container").append(
+          $("<div/>", { "class": "event" }).append(
+            $("<div/>", { "class": "label" }).append(
+              $("<a/>", { "href": "https://github.com/" + field.actor.display_login, "target": "_blank" }).append(
+                $("<img>", { "src": field.actor.avatar_url })
               )
-            ).append(
-              $("<div/>", { "class": "ui small message" }).append(
-                $("<div/>", { "class": "header" }).append(
-                  $("<a/>", { "href": link, "style": "text-decoration: none;" }).append(
-                    $("<h3/>").text(header)
-                  )
+            )
+          ).append(
+            $("<div/>", { "class": "content" }).append(
+              $("<div/>", { "class": "summary" }).append(
+                $("<a/>", { "href": "https://github.com/" + field.actor.display_login, "target": "_blank" }).text(field.actor.display_login)
+              ).append(" " + eventType + " ").append(
+                $("<a/>", { "href": "https://github.com/" + field.repo.name, "target": "_blank" }).text(field.repo.name)
+              ).append(
+                $("<div/>", { "class": "date", "style": "font-size: 100%;" }).append(
+                  " " + timeSince(new Date(field.created_at))
                 )
               ).append(
-                content ? replaceURLsWithLinks(new showdown.Converter().makeHtml(content)) : ""
+                $("<div/>", { "class": "ui small message" }).append(
+                  $("<div/>", { "class": "header" }).append(
+                    $("<a/>", { "href": link, "style": "text-decoration: none;", "target": "_blank" }).append(
+                      $("<span/>", { "class": "iconify", "data-icon": icon, "style": "color: " + iconColor })
+                    ).append(
+                      $("<span/>").text(" " + header)
+                    )
+                  )
+                ).append(
+                  content ? converter.makeHtml(content) : ""
+                )
               )
-            )
-          ))
-      ).append(
-        $("<div/>", { "class": "ui divider" })
-      );
+            ))
+        ).append(
+          $("<div/>", { "class": "ui divider" })
+        );
+      });
+
+      if (loadMoreButton) {
+        $(".feed-container").append(
+          $("<button/>", { "class": "fluid ui button", "id": "loadMore" }).text("Load more")
+        )
+      }
 
       $(window).trigger("resize");
-    });
-  })
-}());
+
+      $("#loadMore").click(function () {
+        pageToLoad++;
```

Comment:
this one is a bit puzzling, it's not a big deal, but if you learn to avoid it is better.
In short, if we make the function able to handle this entirely on its own, it's better because the code is easier to maintain.
For example, you could pass an additional parameter to the options, eg: `addPage`, which if `true` increments `window.__githubApiPage`, this way this counter would be entirely handled by the function and all the logic would be encapsulated in it, which would make it easier to reuse and maintain.